syntax = "proto3";

package Ydb.Debug.V1;
option java_package = "com.yandex.ydb.debug.v1";

import "ydb/public/api/protos/ydb_debug.proto";

service DebugService {
    rpc PingPlainGrpc(Debug.PlainGrpcRequest) returns (Debug.PlainGrpcResponse);
    rpc PingGrpcProxy(Debug.GrpcProxyRequest) returns (Debug.GrpcProxyResponse);
    rpc PingKqpProxy(Debug.KqpProxyRequest) returns (Debug.KqpProxyResponse);
    rpc PingSchemeCache(Debug.SchemeCacheRequest) returns (Debug.SchemeCacheResponse);
    rpc PingTxProxy(Debug.TxProxyRequest) returns (Debug.TxProxyResponse);
}
